### DISCUSSION TITLE
what guide to follow to build like this efi

### DISCUSSION THREAD

**Discussion by UsetUset2244** [kalkmann/Legion-5600H-Hackintosh#4] [General]
i wanna know what guide should i follow

**Comment by kalkmann** [comment]
> i wanna know what guide should i follow

Sorry for the late answer:

**Dortania Guide**

https://dortania.github.io/OpenCore-Install-Guide/prerequisites.html

https://dortania.github.io/OpenCore-Install-Guide/AMD/zen.html#starting-point

**OC-Little Guides**

https://github.com/5T33Z0/OC-Little-Translated

**Help Forum**

https://forum.amd-osx.com/

https://discord.gg/EfCYAJW
